ee/server/service/mdm.go

```diff
@@ -1274,6 +1274,22 @@ func (svc *Service) ListABMTokens(ctx context.Context) ([]*fleet.ABMToken, error
 	return tokens, nil
 }
 
+func (svc *Service) CountABMTokens(ctx context.Context) (uint32, error) {
+	// Authorizing using the more general AppConfig object because:
+	// - this service method returns a count, which is not sensitive information
+	// - gitops role, which needs this info, is not authorized for AppleBM access (as of 2024/12/02)
+	if err := svc.authz.Authorize(ctx, &fleet.AppConfig{}, fleet.ActionRead); err != nil {
```

Comment:
Makes sense. In other places we granted read to gitops, e.g. `mdm_config_profile`:

```
# Global admins, maintainers and gitops can read and write MDM config profiles.
allow {
  object.type == "mdm_config_profile"
  subject.global_role == [admin, maintainer, gitops][_]
  action == [read, write][_]
}
```

But it's probably best we keep the read authorizations of this role to a minimum.